calicoctl/calico_ctl/node.py

```diff
@@ -87,6 +87,9 @@
 CALICO_NETWORKING_ENV = "CALICO_NETWORKING"
```

Comment:
Hmm, I'm very surprised that this was added as an environment variable for calicoctl instead of a commandline flag. @caseydavenport @tomdee Any thoughts on why we chose to do it this way?